features/db_effect.feature

```diff
@@ -0,0 +1,32 @@
+    Scenario: Data base effect implementation
+
+Feature: Data base effect validation
+        Given I get targets list
+        When I should see the 3 numerical targets in the list
+        And I should see each target number should be in the range of citizens list
+        And I should see only targets are relevant to alive citizens
```

Comment:
What if Data base card used when there are only 2 citizens alive?

Reply:
Resolved by discussion.
DB effect can be used on the dead citizens and yourself now.